content/courses/ada_2022_whats_new/chapters/delta_aggregates.rst

```diff
@@ -0,0 +1,97 @@
+.. include:: ../../global.txt
+
+Delta Aggregates
+================
+
+.. note::
+
+    Delta aggregates are supported by
+
+    * GNAT Community Edition 2019
+    * GCC 9
+
+Sometimes you need to create a copy of an object with a few
+modifications. Before Ada 2022 it would involve a dummy object
+declaration or an aggregate with associations for each property.
+The dummy object approach doesn't work in contract aspects.
+A limited component won't work with dummy object approach neither.
+On the other side re-listing properties in an aggregate could be
+too hard. So, in Ada 2022, you can use a `delta aggregates` instead.
+
+Delta aggregate for records
+---------------------------
+
+The delta aggregate for a record type looks like this:
+
+.. code-block:: ada
```

Comment:
It would be great if you could convert this code block and the next one into code snippets using the widget.